pennylane/operation.py

```diff
@@ -1385,6 +1385,41 @@ def compute_qfunc_decomposition(*args, **hyperparameters) -> None:
 
         raise DecompositionUndefinedError
 
+    @classproperty
+    def resource_param_keys(self) -> tuple:
```

Comment:
This will be renamed `resource_keys`, and changed from a `tuple` to a `set` in a separate PR later on.

Reply:
NVM might as well do it here.